### PR TITLE
Increase consistency and update diagrams

### LINKED ISSUE #100: UML for Logic
Update sequence diagram for logic class for DG

### PR DESCRIPTION
**Pull request by starchypotatocode**
Fix #100

### CODE REVIEW

File: src/main/java/hallpointer/address/logic/commands/AddSessionCommand.java [hunk 1]
```diff
@@ -49,6 +49,7 @@ public class AddSessionCommand extends Command {
     public AddSessionCommand(Session session, Set<Index> memberIndexes) {
         requireNonNull(session);
         requireNonNull(memberIndexes);
+        assert !memberIndexes.isEmpty();
```
Comment (yuvrajaryan) [inline src/main/java/hallpointer/address/logic/commands/AddSessionCommand.java:52]: Should we throw an error instead for defensive coding?

**Review by yuvrajaryan — APPROVED** [review]
LGTM. Need to check failing build.

**Comment by yuvrajaryan** [comment]
LGTM

File: src/test/java/hallpointer/address/model/session/SessionTest.java
Comment (taggyhan) [inline src/test/java/hallpointer/address/model/session/SessionTest.java:1]: nice catching this test cases deets

**Pull request merged**